components/automate-chef-io/content/docs/applications-dashboard.md

```diff
@@ -103,9 +103,7 @@ See the [Habitat CLI documentation](https://www.habitat.sh/docs/habitat-cli/#hab
 
 ### Marking a Service as Disconnected
 
-Configure the threshold for marking a service as disconnected through either Chef Automate or the Chef Automate API.
```

Comment:
im not sure why these were added, this is not possible from the UI yet

Reply:
Weird. We must have thought it worked.